tests/ui/test_display.py

```
@@ -0,0 +1,93 @@
+        printed_table = mock_console.print.call_args[0][0]
+        self.assertIsInstance(printed_table, Table)
+        
+        # Check table content
+        self.assertEqual(len(printed_table.rows), 3)
+        self.assertEqual(printed_table.rows[0].cells[1], 'Feed 1')
+        self.assertEqual(printed_table.rows[1].cells[1], 'Feed 2')
+        self.assertEqual(printed_table.rows[2].cells[1], 'Feed 3')
+
+    @patch('src.ui.display.console')
+    @patch('src.ui.display.feedparser.parse')
+    def test_display_latest_articles(self, mock_parse, mock_console):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_display_latest_articles`

Add return type annotation: `None`

(ANN201)